components/tabs/stories/tabs.test.js

```diff
@@ -76,7 +51,49 @@ export const TabsGroups = Variants({
 			testHeading: "Compact overflow",
 			orientation: "overflow",
 			isCompact: true,
-			isQuiet: true,
+		},
+		{
+			testHeading: "Hover colors",
+			withStates: false,
+			content: [
+				{
+					label: "Hovered unselected tab",
+					icon: "Folder",
+					isHovered: true,
+				},
+				{
+					label: "Hovered selected tab",
+					icon: "Image",
+					isSelected: true,
+					isHovered: true,
+				},
+				{
+					label: "Unhovered unselected tab",
+					icon: "File",
+				},
+				{
+					label: "Disabled tab",
+					icon: "Archive",
+					isDisabled: true,
+				}
+			]
+		},
+		{
+			testHeading: "Disabled tabs",
```

Comment:
Since we have disabled tabs captured in the other test cases, do you think we should still have this separate test case? Is it redundant?